enterprise/api_client/lms.py

```diff
@@ -229,6 +229,20 @@ def get_course_enrollment(self, username, course_id):
 
         return result
 
+    def is_enrolled(self, username, course_id):
```

Comment:
Can we move towards standardizing on `course_run_id` when referring to the unique string identifier for a course run?

Reply:
Yes, good idea.

Reply:
Ah I forgot to change it in this function; only changed things in the `RouterView`. Updating...